urbanairship-accengage-react-native/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "urbanairship-accengage-react-native",
+  "version": "7.0.0",
+  "description": "Airship accengage module for React Native apps.",
+  "main": "./js/index.js",
+  "author": "Airship",
+  "homepage": "https://github.com/urbanairship/react-native-module",
+  "keywords": [
+    "react",
+    "react-native",
+    "react-component",
+    "ios",
+    "android",
+    "urbanairship",
+    "accengage"
+  ],
+  "devDependencies": {
+    "react": "^16.9.0",
+    "react-native": "^0.61.5"
```

Comment:
Should this also have dependencies on our main module?

Reply:
YES 👍